src/features/rewards/profile/index.tsx

```diff
@@ -109,6 +117,18 @@ export default class Profile extends React.PureComponent<Props, {}> {
               <StyledVerifiedText>
                 {getLocale('unVerifiedPublisher')}
               </StyledVerifiedText>
+              {
+                !publisherRefreshed ?
```

Comment:
we don't need new prop, we can use `onRefreshPublisher` to determinate if we should show it or not

Reply:
The problem I'm having with that is that we also need value to determine whether spinner is shown. Open to ideas.

Reply:
talked it over on the call, so all good here